src/selection_method.py

```diff
@@ -83,14 +83,16 @@ def get_winners(self, population: List[Individual], k: int, fitness: Callable[[I
         if len(population) == 0:
             return winners
 
+        # TODO: Check repeated winners?
+
         for i in range(k):
-            random.shuffle(population)
-            best = population[0]
+            best = population[random.randint(0, len(population))]
```

Comment:
guardate el len(population) en alguna variable porque len es O(n) en python porque literal recorre el array para ver cuanto mide

Reply:
tambien me fije recien y randint incluye los dos extremos asi que podria irse out of bounds, habria que cambiarlo en el de abajo tambien